src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.html

```diff
@@ -36,7 +45,11 @@
                 </div>
             </ng-container>
             <div class="navigation-item">
-                <button class="btn btn-secondary w-100" (click)="changeExercise(exerciseIndex + 1, false)">
+                <button *ngIf="!overviewPageOpen" class="btn btn-secondary w-100" (click)="changeExercise(exerciseIndex + 1, false)">
+                    <span aria-hidden="true">&raquo;</span>
+                    <span class="sr-only">Next</span>
+                </button>
+                <button *ngIf="overviewPageOpen" class="btn btn-secondary w-100" (click)="changeExercise(0, false)">
                     <span aria-hidden="true">&raquo;</span>
                     <span class="sr-only">Next</span>
                 </button>
```

Comment:
I think this logic must be carried into the class because they are same except function call

Reply:
Which buttons to you mean with the same? @iozbeyli

Reply:
Two buttons from 48 to 55. You can just define a function like nextExercise and go to index 0 if condition is true

Reply:
done